src/main/resources/logback.xml

```diff
@@ -33,6 +29,21 @@
         </encoder>
     </appender>
 
+    <appender name="json" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>DEBUG</level>
+        </filter>
+        <file>/tmp/dag.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>/tmp/dag.%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+            <maxFileSize>500MB</maxFileSize>
+            <totalSizeCap>20GB</totalSizeCap>
+            <maxHistory>60</maxHistory>
```

Comment:
20 GB / 500 MB would give you 40 so it's never reached I would decrease all of those properties to keep  only amount needed for filebeat to catch up